build_docker.sh

```diff
@@ -1 +0,0 @@
-docker build -t mottosso/qt.py27 -f Dockerfile-py2.7 .
```

Comment:
Could we keep this? It's such that a user could build it without remembering the exact label and name of Dockerfile.